ArduCopter/mode_auto.cpp

```diff
@@ -1916,6 +1916,9 @@ void ModeAuto::do_winch(const AP_Mission::Mission_Command& cmd)
         case WINCH_RATE_CONTROL:
             g2.winch.set_desired_rate(cmd.content.winch.release_rate);
             break;
+        case WINCH_DELIVER:
+            g2.winch.deliver_package();
```

Comment:
In this case I think we would also want a "verify" method so the command is only marked as complete once the package has been dropped.